src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppInstanceStatus.java

```diff
@@ -74,7 +74,19 @@ public DeploymentState getState() {
 
 	@Override
 	public Map<String, String> getAttributes() {
-		return new HashMap<>(); // TODO
+		Map<String, String> attributes = new LinkedHashMap<>();
```

Comment:
It's part of the deployer API.
Try runtime apps in the shell

Le 22 août 2016 12:51 PM, "Ilayaperumal Gopinathan" <
notifications@github.com> a écrit :

> In src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/
> CloudFoundryAppInstanceStatus.java
> <https://github.com/spring-cloud/spring-cloud-deployer-cloudfoundry/pull/79#discussion_r75656372>
> :
>
> > @@ -74,7 +74,19 @@ public DeploymentState getState() {
> >
> >  	@Override
> >  	public Map<String, String> getAttributes() {
> > -		return new HashMap<>(); // TODO
> > +		Map<String, String> attributes = new LinkedHashMap<>();
>
> LGTM. But I don't see this method being used anywhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spring-cloud/spring-cloud-deployer-cloudfoundry/pull/79/files/a73fade399d50b444ae7b0d85322bf50389c2450#r75656372>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATIlq5t3KdEFMcoH1RmAFt2Adcs9Eptks5qiX8ogaJpZM4Jptk8>
> .
>